slides/bitcoin_applications.tex

```diff
@@ -0,0 +1,491 @@
+%\documentclass[handout]{beamer}
+
+% Video Meta Data
+\title{Bitcoin, Blockchain and Cryptoassets}
+\subtitle{Bitcoin Applications}
+\author{Prof. Dr. Fabian Schär}
+\institute{University of Basel}
+
+% Config File
+\input{../config/config.tex}
+
+% Adiitional Files
+\input{../config/solidity.tex} % specification of solidity color coding
+\input{../config/json.tex} % specification of JSON color coding
+\input{../config/bitcoin_symbol.tex} % definition of bitcoin symbol
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\begin{document}
+
+\thispagestyle{empty}
+\begin{frame}[noframenumbering]
+	\titlepage
+\end{frame}
+
+%%%
+
+\begin{frame}{External Data}	
+	\begin{itemize}
+		\item<1 ->\textbf{Native On-Chain Data}
+		\begin{itemize}
+			\item<1 ->Data that is stored on-chain and fully secured by the consensus protocol (e.g. Bitcoin trx)\\
+			\vspace{0.25em}$\rightarrow$ Automated evaluation and verification
+		\end{itemize}
+		\vspace{1em}
+		\item<2 ->\textbf{Added Off-Chain Data}
+		\begin{itemize}
+			\item<2 -> Data that is stored on-chain but not secured by the consensus protocol (e.g. course certificates)\\
+			\vspace{0.25em}$\rightarrow$ Automated evaluation, subject to data quality
+		\end{itemize}
+		\vspace{1em}
+		\item<3 -> \textbf{Off-Chain Data}
+		\begin{itemize}
+			\item<3 -> Data that is NOT directly stored on the Blockchain (e.g.~weather, results of a football game)\\
+			\vspace{0.25em}$\rightarrow$ Requires trustworthy data providers (oracles)
+		\end{itemize}
+	\end{itemize}
+\end{frame}
+
+%%%
+
+\begin{frame}{External Data and the Oracle Poblem}
+	\begin{columns}
+		\begin{column}{0.6\textwidth}
+			\begin{itemize}[<+->]
+				\item Example: Tracking ownership of a parcel
+				\item On-Chain representation of parcel with unique token
+				\item Reproducible crypto-anker (e.g.~QR-code) is problematic
+			\end{itemize}
+		\end{column}
+		\begin{column}{0.4\textwidth}
+			\includegraphics[width=4cm]{../assets/images/parcel.png}
+		\end{column}
+	\end{columns}
+\end{frame}
+
+%%%
+
+\begin{frame}{Oracles on Bitcoin}
+	\textbf{Example:} Alice and Bob want to bet on some event.\\
+	\vspace{0.25cm}
+	\uncover<2 ->{\textbf{Three possibilities:}}
+	\begin{itemize}
+		\vspace{0.1cm}
+		\item<2->[1.] Both parties transfer their stake to the address of the oracle. The oracle then sends the prize to the winning party after the event has taken place.
+		\begin{itemize}
+			\item<2-> Problem: Control over funds lies solely with the oracle.
+		\end{itemize}
+		\vspace{0.2cm}
+		\item<3->[2.] They create a 2-of-3 multisig transaction. Alice, Bob and the oracle each hold a key.
+		\begin{itemize}
+			\item<3-> Control distributed and Alice and Bob can overrule the oracle.
+		\end{itemize}
+		\vspace{0.2cm}
+		\item<4->[3.] More than one oracle and use m-of-n multisig transaction. 
+		\begin{itemize}
+			\item<4-> Agents hold $\frac{m}{2}$ keys and m-1 oracles hold one key each.
```

Comment:
Agents "each" hold m/2 keys?

E.g. m = 4
- each agent holds 2 keys (2*2)
- each oracle holds 1 key (3*1)
- total of 7 keys
- 4-of-7 multisig